data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	He mentions how this is due to him wishing to enter the scientific branch of the Heliarch ranks, so the exact same benefits his research is giving him wouldn't apply if he was looking forward to entering a branch more focused on military operations.`
+				goto bunk
+			label work
+			`	"Larger than most, the Kimek population is. Struggle to properly distribute medication or other chemicals, even worlds richer than my homeworld do," he says. "Seeking to create an independent network to provide better distribution, I am."`
+			`	He jokes about how it's silly that his species has such incredible advancements in their alimentary transport network, it's still difficult for most of the population to find medicine for what is translated as "the common cold" by the device. He claims that, were he on the Heliarch council, he could directly propose changes and solutions to such problems.`
+			label bunk
+			`	He tells you a bit more about his work here, and how it was working with Saryds, and the cultural exchange. Though he seems to have garnered some fond memories of his time here, he still hasn't gotten used to the wind, you notice when you're both heading to your ship.`
+			`	You show him to his bunk, and prepare for the journey to <destination>.`
+	on complete
+		"coalition jobs" ++
+		event "chikee break 3" 78
+		payment 130000
+		conversation
+			`Chikee hands you 130,000 credits when you land.`
+			`	"As always, for transporting me grateful I am, Captain," he says, as he heads off to meet with the members of House Idriss.`
+
+
+
+event "chikee break 3"
+
+
+
+mission "Kimek Student 4"
+	name "Transport Chikee"
+	description "Chikee, the young Kimek who you've transported many times, has finally become a candidate for elevation to the rank of Heliarch. Bring him to <destination> by <date>."
+	deadline
+	landing
+	passengers 1
+	to offer
+		has "event: chikee break 3"
+	source "Ablub's Invention"
+	destination "Ring of Wisdom"
+	on offer
+		conversation
+			`You're contacted by Chikee when you land.`
+			`	"Captain <last>! Good news, and another job for you, I have," he says, asking that you meet him in the spaceport.`
+			`	You head there, and he's already with his bags, prepared to board your ship. "Helped me a lot, House Idriss has. An audience with the Heliarch, on <destination> I have!" he says.`
+			`	He says that he needs to be there by <date>, and wastes no time as he immediately heads into your ship.`
+				accept
+	on complete
+		"coalition jobs" ++
+		payment 300000
+		conversation
+			`Chikee hands you <payment> before leaving your ship. Heliarch agents are waiting for him, and receive him in a bit of a ceremony.`
+			`	You spend a few hours browsing some shops here, and have some food once you find an establishment that serves something appetizing to humans.`
+			`	Ringworlds have no day or night, but looking at your ship's internal clock, it is almost night time when Chikee returns to your ship, in a hurry.`
```

Comment:
some planets, specifically a few in Coalition space itself, touch on how the day-night cycle works on them, I just added this bit here to be something like that, only in a mission rather than a planet description.

Reply:
Fair enough.